component/appcat_apiserver.jsonnet

```diff
@@ -78,6 +79,18 @@ local extraDeploymentArgs =
 local apiserver = loadManifest('aggregated-apiserver.yaml') {
   metadata+: {
     namespace: apiserverParams.namespace,
+    annotations+: {
+      'metadata.appcat.vshn.io/enabled-service-hash': std.md5(
```

Comment:
Why not put the whole `std.md5` call into `vars` as a function?

Reply:
indeed why not :D